src/Evolution/Initialization/CoordinateMaps.hpp

```diff
@@ -0,0 +1,68 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <utility>
+
+#include "DataStructures/Tensor/IndexType.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/Identity.hpp"
+#include "Domain/CoordinateMaps/Tags.hpp"
+#include "Domain/Creators/Tags/Domain.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/ElementMap.hpp"
+#include "Domain/Structure/Element.hpp"
+#include "Domain/Structure/ElementId.hpp"
+#include "Domain/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace amr::projectors {
+/// \brief Initialize/update items related to coordinate maps after an AMR
+/// change
+template <size_t Dim>
+struct CoordinateMaps {
+  using return_tags = tmpl::list<::domain::Tags::ElementMap<Dim, Frame::Grid>,
+                                 ::domain::CoordinateMaps::Tags::CoordinateMap<
+                                     Dim, Frame::Grid, Frame::Inertial>>;
```

Comment:
This initializes the tags added by `evolution::dg::Initialization::Domain` in `DgDomain.hpp`, so move this struct into the same file and name `ProjectDomain` or something like that. When I added support for AMR it helped a lot to have the AMR mutator right next to the initialization mutator. Same for the time stepping mutator.